packages/frontend/core/package.json

```diff
@@ -18,6 +18,8 @@
     "@affine/track": "workspace:*",
     "@blocksuite/affine": "0.17.19",
     "@blocksuite/icons": "2.1.69",
+    "@capacitor/app": "^6.0.1",
```

Comment:
I think we should avoid referencing the capacitor package in core

Reply:
but `popupWindow` is implemented in core. Should we wrap a function in `@affine/ios` and import from it?